### PR TITLE
added pvs-studio artifacts to gitignore

### PR DESCRIPTION
**Pull request by p01arst0rm**
added pvs-studio to gitignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -65,6 +65,9 @@
 *.vcxproj.user
 enc_temp_folder/*
 CMakeSettings.json
+*PVS-Studio*
+*pvs-studio*
+*PVS*
```
Comment (Nekotekina) [inline .gitignore:70]: Does it work if you keep only ``*pvs*``?

Comment (p01arst0rm) [reply .gitignore:70]: id imagine so.. :*

Comment (Nekotekina) [reply .gitignore:70]: You mean, it doesn't ignore all files?

Comment (p01arst0rm) [reply .gitignore:70]: no it does, i just didnt think about it that way

Comment (elad335) [reply .gitignore:70]: *PVS* may be a too generic pattern, especially in RPCS3 where we use abbreviations alot, is it possible for it to be more specific?

Comment (p01arst0rm) [reply .gitignore:70]: just double checked, it needs to check for all files including `PVS-Studio`. it creates lock files as it traverses the codebase, which should be ignored

Comment (p01arst0rm) [reply .gitignore]: PVS/* allows you to create a PVS workspace inside of the project and have that ignored too.

**Pull request merged**